pkg/providers/cloudstack/testdata/cluster_main.yaml

```diff
@@ -125,9 +125,5 @@ spec:
     device: "/dev/vdb"
     filesystem: "ext4"
     label: "data_disk"
-  symlinks:
-    /var/lib/: /data-small/var/lib
```

Comment:
can we leave this symlink for etcd, and fix the trailing `/` ?

Reply:
I think having a symlink for this kind of system folder is risky.

Reply:
the idea is trying to redirect the etcd files into an attched disk offering.
/var/lib is the only settings that we tested working. If we put /var/lib/etcd , etcd will complain and refuse to start.

also, we have scripts that works with it:
```
{{- range $dir, $target := .cloudstackEtcdSymlinks}}
    - >-
      if [ ! -L {{$dir}} ] ;
        then
          mv {{$dir}} {{$dir}}-$(tr -dc A-Za-z0-9 < /dev/urandom | head -c 10) ;
          mkdir -p {{$target}} && ln -s {{$target}} {{$dir}} ;
        else
          echo "{{$dir}} already symlnk" ;
      fi
{{- end}}
```